.env.example

```diff
@@ -1 +1,5 @@
 VITE_POLYGON_MIGRATION_TRANSACTION_FEE=0.05
+VITE_MIGRATION_GUIDE_URL=https://arkvault.io/docs
+VITE_MIGRATION_ADDRESS=DNBURNBURNBURNBRNBURNBURNBURKz8StY
+VITE_MIGRATION_NETWORK=ark.mainnet
```

Comment:
can you change it to default to `ark.devnet` so it's in line with the default burn address as well